egs/wsj/s5/steps/nnet3/decode.sh

```diff
@@ -80,7 +80,7 @@ thread_string=
 [ $num_threads -gt 1 ] && thread_string="-parallel --num-threads=$num_threads"
 
 mkdir -p $dir/log
-[[ -d $sdata && $data/feats.scp -ot $sdata ]] || split_data.sh $data $nj || exit 1;
+[[ -d $sdata && $data/feats.scp -ot $sdata ]] || split_data.sh --per-utt $data $nj || exit 1;
```

Comment:
Was the addition of this --per-utt flag intended to be temporary?
Until issue #778 is addressed, this is not safe to commit as it would interfere with jobs that do require splitting per-speaker.